app/src/main/res/layout/number_picker_dialog_layout.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <NumberPicker
+        android:id="@+id/numberPicker"
+        android:layout_width="85dp"
+        android:layout_height="156dp"
+        android:layout_alignParentEnd="true"
+        android:layout_margin="20dp" />
+
+    <TextView
+        android:id="@+id/textView"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_alignParentStart="true"
+        android:layout_marginStart="20dp"
+        android:layout_marginTop="85dp"
+        android:text="@string/days_before_expiry"
+        android:textColor="@android:color/black"
```

Comment:
This should probably be something like
```
android:textColor="?android:attr/textColorSecondary"
```
so it doesn't look all wrong:
![image](https://user-images.githubusercontent.com/1885159/141515643-08f6bebc-38f2-446a-89d8-d2ee1efa6846.png)

(Or maybe you don't need to set textColor at all, needs testing)